blazor/pager/data-binding.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Data Binding in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about working with data in Syncfusion Blazor Pager component and much more.
```

Comment:
Checkout and learn here all about working Syncfusion Blazor Pager component with both local or remote data collection and how it splits into sectioned pages.